src/schema/mutation/editResource.mutation.ts

```diff
@@ -159,7 +159,7 @@ const checkFieldPermission = (
           )
         );
       }
-      if (!get(field, 'permissions.canSee', []).find((p) => p.equals(role))) {
+      if (!get(field, 'permissions.canSee', []).find((p) => p === role)) {
```

Comment:
```suggestion
      if (!get(field, 'permissions.canSee', []).find((p) => p.toString() === role)) {
```

This guarantees that p is a string and not an oid